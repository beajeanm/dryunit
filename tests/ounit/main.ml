open OUnit2

let () =
  run_test_tt_main (
    "All tests" >::: [
      "Secondary.test_in_secondary" >:: Secondary.test_in_secondary;

      "Tests.test1" >:: Tests.test1;
      "Tests.test2" >:: Tests.test2;

    ]
  )
