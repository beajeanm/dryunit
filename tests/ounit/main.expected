Ran: 3 tests
